### DISCUSSION TITLE
Virgin media blocked connections

### DISCUSSION THREAD

**Discussion by steven86m** [amidaware/tacticalrmm#2019] [General]
Hi we have some users that have tactical installed on their Pc's/Laptops but when they are working from home I cannot connect to them if they are on Virgin Media, has anyone got a solution to this?

**Comment by P6g9YHK6** [comment]
seems impossible as all the connection are over https and going to your own server so the only case would be that your ip is blacklisted by them and you shoud either change server IP or contact them to see why
but to make sure of what happens currently you should do a tcp check over port 443 from one laptop to the rmm server to see if the traffic is indeed not being routed somewhere.
